keyboards/rephlex/2k/rules.mk

```diff
@@ -0,0 +1,7 @@
+# Path: keyboards/rephlex/2k/rules.mk
+QUANTUM_LIB_SRC += analog.c
+SRC += matrix.c lut.c scanfunctions.c
+
+CUSTOM_MATRIX = lite
+
+OPT = 2
```

Comment:
Not really a fan of this. 

Is there a specific reason for setting this, at all?